extensions/Overview.html

```diff
@@ -568,38 +551,38 @@ <h3 class="heading settled" data-level="4.2" id="C.pCAL"><span class="secno">4.2
    <table>
     <tbody>
      <tr>
-      <th>Calibration name:
-      <td>1-79 bytes (character string)
+      <th>Calibration name: 
+      <td>1-79 bytes (character string) 
      <tr>
-      <th>Null separator:
-      <td>1 byte
+      <th>Null separator: 
+      <td>1 byte 
      <tr>
-      <th>Original zero (x0):
-      <td>4 bytes (signed integer)
+      <th>Original zero (x0): 
+      <td>4 bytes (signed integer) 
      <tr>
-      <th>Original max  (x1):
-      <td>4 bytes (signed integer)
+      <th>Original max  (x1): 
+      <td>4 bytes (signed integer) 
      <tr>
-      <th>Equation type:
-      <td>1 byte
+      <th>Equation type: 
+      <td>1 byte 
      <tr>
-      <th>Number of parameters:
-      <td>1 byte
+      <th>Number of parameters: 
+      <td>1 byte 
      <tr>
-      <th>Unit name:
-      <td>0 or more bytes (character string)
+      <th>Unit name: 
+      <td>0 or more bytes (character string) 
      <tr>
-      <th>Null separator:
-      <td>1 byte
+      <th>Null separator: 
+      <td>1 byte 
      <tr>
-      <th>Parameter 0 (p0):
-      <td>1 or more bytes (<a href="#DR.Floating-point-values">ASCII floating-point</a>)
+      <th>Parameter 0 (p0): 
+      <td>1 or more bytes (<a href="#DR.Floating-point-values">ASCII floating-point</a>) 
      <tr>
-      <th>Null separator:
-      <td>1 byte
+      <th>Null separator: 
+      <td>1 byte 
      <tr>
-      <th>Parameter 1 (p1):
-      <td>1 or more bytes (<a href="#DR.Floating-point-values">ASCII floating-point</a>)
+      <th>Parameter 1 (p1): 
+      <td>1 or more bytes (<a href="#DR.Floating-point-values">ASCII floating-point</a>) 
```

Comment:
Was adding the trailing spaces intentional?

Reply:
I have no idea why the trailing whitespace appeared. It is only on the bikeshed-generated output. My editor is set to strip trailing whitespace (which sometimes gives diff problems)